cmd/nodepool/create.go

```diff
@@ -68,6 +77,9 @@ func NewCreateCommand() *cobra.Command {
 	cmd.Flags().StringVar(&opts.RootVolumeType, "root-volume-type", opts.RootVolumeType, "The type of the root volume (e.g. gp2, io1) for machines in the NodePool")
 	cmd.Flags().Int64Var(&opts.RootVolumeIOPS, "root-volume-iops", opts.RootVolumeIOPS, "The iops of the root volume when specifying type:io1 for machines in the NodePool")
 	cmd.Flags().Int64Var(&opts.RootVolumeSize, "root-volume-size", opts.RootVolumeSize, "The size of the root volume (min: 8) for machines in the NodePool")
+	cmd.Flags().StringVar(&opts.KubevirtMemory, "kubevirt-memory", opts.KubevirtMemory, "In KubeVirt platform - the amount of memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)")
```

Comment:
here the `kubevirt-*` prefix is probably correct, but I think this is a part of the commandline tool which I did not address in the `None` platform refactoring. We probably want sub-commands for this too. I think this refactoring can be done independent of this PR @enxebre what do you think?